app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with(model: [@group, @expense], local: true) do |f| %>
+  <div class="header">
+    <%= link_to group_expenses_path(@group) do %>
+      <%= image_tag 'back_arrow.svg' %>
+    <% end %>
+    <p>Add new transaction</p>
+    <div class="actions">
+      <%= f.submit "Save", class: "header-btn" %>
+    </div>
+  </div>
+
+  <div class="form">
+    <div class="field">
+      <%= f.text_field :name, autofocus: true, placeholder: 'Transaction name' %>
+    </div>
+
```

Comment:
- [ ] Amazing job working on the requirements so far, I think you made huge efforts on the project but you still have one small thing to proceed, so here when you add a new transaction I think you added the name and the amount inputs but it is mentioned to add an input for a category so you let the user be able to choose the category that he spent this specific transaction on it. so I think it would be great if you add it 🤩✨🚀💯✅


![image](https://github.com/Simpleshaikh1/BudgetBuddy/assets/58553711/ffdea763-9c47-4478-9bf0-c80969f98820)